src/services/github.ts

```diff
@@ -29,12 +30,16 @@ export default class GithubService {
         set a token on the 'password' field in ~/.netrc for api.github.com`);
       }
 
-      this.octokit = new RetryableOctokit({
+      this.octokit = new RetryableThrottledOctokit({
         auth: token,
-        retry: {
-          // By default the doNotRetry setting includes 403, which means we face secondary rate limits
-          doNotRetry: [400, 401, 404, 422],
-        }
```

Comment:
I dropped this since the new plugin should now handle `403` errors for us.

Reply:
Nice!